frontend/__tests__/public/co-fetch.spec.js

```diff
@@ -1,6 +1,16 @@
-import { shouldLogout } from '../../public/co-fetch';
+import { coFetch, shouldLogout, validateStatus, RetryError } from '../../public/co-fetch';
 
 describe('coFetch', () => {
+  const json = async () => ({
+    details: {
+      group: 'quota.openshift.io',
+      kind: 'clusterresourcequotas',
+    },
+  });
+  const emptyHeaders = new Headers();
+  const headers = new Headers();
+  headers.set('content-type', 'json');
```

Comment:
I know this is just test code, but this should probably be `application/json`.

```suggestion
  headers.set('content-type', 'application/json');
```

Reply:
updated